modules/insticatorBidAdapter.js

```diff
@@ -190,17 +205,59 @@ function buildDevice(bidRequest) {
   return device;
 }
 
+function _getCoppa() {
+  return config.getConfig('coppa') === true ? 1 : 0;
```

Comment:
whose config does this refer?

Reply:
This is the common config from the prebid module.